packages/ui/src/components/SnackBar/SnackStackProvider.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useContext, createContext } from "react";
+import { ToastType } from "../ToastContent";
+import Snackbar from "./Snackbar";
+
+export type ToastProps = {
+  key?: number;
+  title?: string;
+  message?: string;
+  link?: string;
+  duration?: number;
+  type: ToastType;
+  position?: {
+    vertical?: "top" | "bottom";
+    horizontal?: "left" | "right" | "center";
+  };
+};
+
+const MAX_VISIBLE_TOASTS = 3
+
+export type SnackStackContextProps = {
+  toastsPack: ToastProps[];
+  setToastsPack: (toasts: ToastProps[]) => void;
+  addToast: (toast: ToastProps) => void;
+  removeToast: (key: ToastProps["key"]) => void;
+};
+
+const SnackbarContextProvider  = createContext<SnackStackContextProps>({
+  toastsPack: [],
+  setToastsPack: (toasts) => {},
+  addToast: (toast) => {},
+  removeToast: (key) => {}
+});
```

Comment:
This can be simplified a bit
```suggestion
const SnackbarContextProvider  = createContext<SnackStackContextProps | undefined>(undefined);
```